specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-01-preview/images.json

```diff
@@ -0,0 +1,348 @@
+{
+   "swagger": "2.0",
+   "info": {
+      "title": "Cognitive Services API",
+      "description": "Cognitive Services API to support OpenAI Dall-E",
+      "version": "2023-04-01-preview"
+   },
+   "host": "eastus.api.cognitive.microsoft.com",
+   "basePath": "/dalle",
+   "schemes": ["https"],
+   "securityDefinitions": {
+      "Api-Key": {
+        "type": "apiKey",
+        "name": "Api-Key",
+        "in": "header",
+        "description": "Provide your Cognitive Services Azure OpenAI account key here."
+      }
+   },
+   "security": [
+      {
+        "Api-Key": []
+      }
+    ],
+   "parameters": {
+      "apiVersionQueryParameter": {
+        "in": "query",
+        "name": "api-version",
+        "description": "The requested API version.",
+        "required": true,
+        "type": "string"
+      }
+   },
+   "definitions": {
+      "Status": {
+         "description": "The status of the operation.",
+         "type": "string",
+         "enum": [
+            "NotStarted",
```

Comment:
>N [](http://example.com/codeflow?start=13&length=1)

notStarted?

Reply:
See comment above: https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#long-running-operations, the status would always start with a capital letter.